app/src/main/java/org/mozilla/focus/fragment/onboarding/OnboardingFirstScreenCompose.kt

```diff
@@ -0,0 +1,164 @@
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.AnnotatedString
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import mozilla.components.ui.colors.PhotonColors
+import org.mozilla.focus.R
+import org.mozilla.focus.ui.theme.FocusTheme
+import org.mozilla.focus.ui.theme.focusTypography
+
+@Composable
+@Preview
+private fun OnBoardingFirstScreenComposePreview() {
+    FocusTheme {
+        OnBoardingFirstScreenCompose({}, {})
+    }
+}
+
+/**
+ * Displays the first onBoarding screen
+ *
+ * @param onGetStartedButtonClicked when the user clicks on get started button
+ * @param onCloseButtonClick the user clicks on close onBoarding flow
+ */
+@Composable
+fun OnBoardingFirstScreenCompose(
+    onGetStartedButtonClicked: () -> Unit,
+    onCloseButtonClick: () -> Unit
+) {
+    Column(
+        modifier = Modifier
+            .fillMaxWidth()
+            .fillMaxHeight()
+            .background(
+                brush = Brush.linearGradient(
+                    colors = listOf(
+                        colorResource(R.color.biometric_background_fragment_one),
+                        colorResource(R.color.biometric_background_fragment_two),
+                        colorResource(R.color.biometric_background_fragment_three),
+                        colorResource(R.color.biometric_background_fragment_four),
+                        colorResource(R.color.biometric_background_fragment_five),
+                        colorResource(R.color.biometric_background_fragment_six),
+                    ),
+                    end = Offset(0f, Float.POSITIVE_INFINITY),
+                    start = Offset(Float.POSITIVE_INFINITY, 0f)
+                )
+            )
+    ) {
+        Column(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(top = 60.dp, end = 20.dp),
+            horizontalAlignment = Alignment.End
+        ) {
+            CloseButton(onCloseButtonClick)
+        }
+        Column(
+            horizontalAlignment = Alignment.CenterHorizontally,
+            verticalArrangement = Arrangement.Center,
+            modifier = Modifier
+                .fillMaxWidth()
+                .fillMaxHeight()
+                .verticalScroll(rememberScrollState())
+        ) {
+            Image(
+                painter = painterResource(R.drawable.onboarding_logo),
+                contentDescription = LocalContext.current.getString(R.string.app_name),
+                modifier = Modifier
+                    .size(150.dp, 150.dp)
+            )
+            Text(
+                text = stringResource(
+                    id = R.string.onboarding_first_screen_title,
+                    stringResource(id = R.string.app_name)
+                ),
+                style = focusTypography.onboardingTitle,
```

Comment:
Regarding the order of the parameters we try to be consistent about using the parameters in the order of declaration. Here would mean that `style` is the first parameter set.

This direction for the order of parameters should be used for the other composables or normal code that we use also.

Reply:
Done